examples/heterogeneous-hello-world/main.cpp

```diff
@@ -1,3 +1,4 @@
+#if 0
```

Comment:
Maybe we should include a comment here about why the example is being disabled?

Reply:
Commented and moved to `examples/experimental` subdirectory along with `model-viewer`.